apps/web/controllers/pets/edit.rb

```diff
@@ -7,6 +7,7 @@ class Edit
     expose :pet
 
     def call(params)
+      # TODO How I can apply more security here?
       @pet = PetRepository.find(params[:id])
```

Comment:
Nope. I think that the correct way is create a new query... So, we are forced to do not call method in multiple levels.